feat: Add PHP version module

REBELinBLUE opened the pull request:
#### Description
Adds the PHP version to the prompt in directories to `.php` files or `composer.json`

#### Motivation and Context
Since Node, Ruby, Rust, Python and Go are already included

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Screenshots (if appropriate):
<img width="470" alt="Screenshot 2019-08-26 at 22 52 23" src="https://user-images.githubusercontent.com/2143908/63726095-35630280-c854-11e9-937f-6c45ba00fdbc.png">


#### How Has This Been Tested?
- [x] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

This is the first time I have done anything with Rust, I based it on the Ruby module. I am unsure about the actual tests other as most of the existing Ruby ones appear to be ignored? Also not sure if I need to add anything to `tests/Dockerfile`?

#### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

REBELinBLUE commented on src/modules/php.rs:
```diff
@@ -0,0 +1,69 @@
+    }
+}
+
+fn format_php_version(php_version: &str) -> Option<String> {
```
Yeah just noticed this, looking into it now

REBELinBLUE replied:
Sadly not

```$ php -r 'echo PHP_VERSION;'
7.1.31-1+ubuntu16.04.1+deb.sury.org+1z``

I did a fix last night though, looks like I just forgot to push it

REBELinBLUE commented on src/module.rs:
```diff
@@ -21,6 +21,7 @@ pub const ALL_MODULES: &[&str] = &[
     "python",
     "ruby",
     "rust",
+    "php",
```
Ah! I thought there was probably somewhere else it was needed but I couldn't find it

REBELinBLUE commented:
PHP does have a constant with the "extra" stuff stored in, so I will use that to strip it out

```
$ php -r 'echo PHP_EXTRA_VERSION;'
-1+ubuntu16.04.1+deb.sury.org+1
```

REBELinBLUE commented:
Or even

```
$ php -r "echo PHP_MAJOR_VERSION.'.'.PHP_MINOR_VERSION.'.'.PHP_RELEASE_VERSION;"
7.1.31
```

REBELinBLUE commented:
Got it fixed, just trying to fix the Pipeline Mac build

REBELinBLUE commented:
Sorry I haven't had a chance to get the CI build working (the mac build is failing to install PHP) and will be away from the computer until 16th September now so will look then

chipbuster commented:
Oh boy.

@matchai It looks like PRs that haven't been rebased over the new GH Actions framework still use the Azure Pipelines. Will it be worthwhile to rebase these PRs over the new CI before we continue with them?

REBELinBLUE commented:
Ah yeah i've just seen that, I shall see if I can find some time this evening to rebase and try and fix the test

REBELinBLUE commented:
I am now finally back from my trip so I shall get on this again, but it'll be tomorrow evening.

@aequasi at this stage no, but I do plan to look into that later on

REBELinBLUE commented:
Apologizes for the delay on this, I have rebased it, fixed the issues, updated to include the config file, updated the github workflow to install PHP and updated the documentation

REBELinBLUE commented:
No worries

matchai commented:
@all-contributors Please add @REBELinBLUE for code, test, docs. 🎉
